portfolio/src/main/java/com/google/sps/servlets/ContactInformationServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/contact-information")
+public class ContactInformationServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get first name entered in the form.
+    String firstName = getParameter(request, "fname", "");
+    
+    // Get last name entered in the form.
+    String lastName = getParameter(request, "lname", "");
+    
+    // Get the name entered in the form.
+    String email = getParameter(request, "email", "");
+
+    // Reason of contacting
+    boolean academicContact = Boolean.parseBoolean(getParameter(request, "academic-contact", "false"));
+    boolean professionalContact = Boolean.parseBoolean(getParameter(request, "professional-contact", "false"));
+    boolean otherContact = Boolean.parseBoolean(getParameter(request, "other-contact", "false"));
+    String otherContactReason = getParameter(request, "other-contact-reason", "");
+
```

Comment:
`Boolean.parseBoolean(null)` evaluates to `false`, so I am not sure if you really need a `defaultValue` argument in `getParameter`

Reply:
I tried to doing so, but the IDE raises an error. I think one has to feed a `defaultValue` either way for it to work; not sure. But I appreciate the suggestion, it would actually look better that way.

![image](https://user-images.githubusercontent.com/78444591/110989722-a7247380-8348-11eb-80f6-76dd20b26c73.png)